src/protocols/aeternity/libs/AeSdkSuperhero.ts

```diff
@@ -57,4 +57,18 @@ export class AeSdkSuperhero extends AeSdkWallet {
   ): ReturnType<typeof sendTransaction> {
     return super.spend(amount, recipientId, options as any); // TODO: remove type casting once https://github.com/aeternity/aepp-sdk-js/issues/1791 is resolved
   }
+
+  /**
+   * Get Wallet info object
+   * @returns Object with wallet information
+   */
```

Comment:
This comment gives no value :) It's just the function name but written in more readable way.